Tests/Integration/RedisFixture.cs

```diff
@@ -57,44 +59,52 @@ private async Task StartRedisContainer()
                 {
                     Hostname = "localhost"
                 };
-                this.redisPort = Interlocked.Increment(ref uniqueRedisPort);
```

Comment:
[nit] we could let the team know that they don't need the second redis container anymore. Not sure if any docs need to be updated but I would assume no.

Reply:
I checked the docs too, and didn't find an occurrence for this. I will let the team know if we decide to go ahead with one Redis container, and else abandon the PR.